### PR TITLE
Update notes about environment variables

### PR DESCRIPTION
**Pull request by hawkrives**
Since I was just cleaning up stuff in the config files, I decided to go ahead and update the lists of environment variables that we use on our CI environments.

### CODE REVIEW

**Comment by drewvolz** [comment]
Can we update the README to remove the bit about the calendar API key?

File: .circleci/config.yml [hunk 1]
```diff
@@ -38,8 +38,6 @@ x-config:
       name: Embed environment variables into the app
       command: |
         touch .env.js
```
Comment (drewvolz) [inline .circleci/config.yml:40]: Is anything inside this file anymore?

Comment (hawkrives) [reply .circleci/config.yml:40]: No, but … something may come back in the future?

Comment (drewvolz) [reply .circleci/config.yml:40]: Alright. I think one use case we'll have for it is when the map transitions to mapbox. So we can keep it around.

**Review by drewvolz — APPROVED** [review]

**Pull request merged**